docs/docs/hpc-on-k8s/rke.md

```diff
@@ -0,0 +1,87 @@
+# Installing Kubernetes Cluster with RKE
+
+We are using RKE (Rancher Kubernetes Engine) to install a Kubernetes cluster.
+
+## Types of Nodes in Kubernetes Installations
+
+- **rke node (devops node):** This is the node where the RKE utility is installed, responsible for the installation of the Kubernetes cluster on other nodes.
+- **k8s nodes (master/worker nodes):** These are the nodes that are part of the Kubernetes cluster where the installation takes place.
+
+## Prerequisites (Master/Worker)
+
+- Docker must be installed on all nodes of the Kubernetes cluster.
+- Password-less SSH access must be set up from the rke node to all Kubernetes nodes.
+
+```bash
+curl https://releases.rancher.com/install-docker/20.10.sh | sh
```

Comment:
Based on this script and the later use of `snap` I presume this is on Ubuntu? It might be worth noting the Operating System used as a prerequisite or some explanation of the use case

Reply:
this script support all os `debian, raspbian, centos, rhel, sles, rocky, oracleserver, ol, ubuntu, fedora`.
Also updated docs to install kubectl on linux